bundle.sh

```diff
@@ -0,0 +1,10 @@
+rm ./build/umd/bundle.development.js
```

Comment:
Technically since this script is only executed on npm build, these files will never exist. we can probably delete these